imports/ui/pages/deployables/channels/page.html

```diff
@@ -53,7 +53,7 @@ <h4 class="card-header text-muted">Channels
                                     <div class="col-lg-4">
                                         <div class="d-block d-lg-none row-header-sm">ID</div>
                                           <div class="input-group">
-                                            <code class="form-control text-truncate">{{channel.uuid}}"</code>
+                                            <code class="form-control text-truncate">{{channel.uuid}}</code>
                                             <div class="input-group-append">
                                               <button class="btn btn-primary copy-button" type="button" data-clipboard-text="&quot;{{channel.uuid}}&quot;">Copy</button>
```

Comment:
Should it be removed from the copy action as well?
`data-clipboard-text="&quot;{{channel.uuid}}&quot;"`
to
`data-clipboard-text="{{channel.uuid}}"`